src/components/app/Layout/Header/index.jsx

```diff
@@ -7,13 +7,45 @@ import downArrow from '../../../../assets/images/down_arrow.svg'
 import phone from '../../../../assets/images/icon-phone.svg'
 import heart from '../../../../assets/images/icon-heart.svg'
 import cart from '../../../../assets/images/icon-cart.svg'
+import burgerArrow from '../../../../assets/images/burgerMenuArrow.svg'
 import { Link } from 'react-router-dom'
 
 const Header = () => {
+  const [menuActive, setMenuActive] = useState(false)
   const wrapper = cn('wrapper', styles.headerWrapper)
   return (
     <div className={styles.header}>
       <div className={wrapper}>
+        <div className={styles.burgerBlock}>
+          <div onClick={() => setMenuActive(!menuActive)} className={styles.burgerBtn}>
+            <span className={`${styles.burgerBtnOnclick} ${menuActive && styles.burgerBtnClick}`} />
```

Comment:
```suggestion
            <span className={clsx(styles.burgerBtnOnclick, {[styles.burgerBtnClick]: menuActive )} />
```

а лучше вынести отдельно 
```
const burgerBtnClassname = clsx(styles.burgerBtnOnclick, {[styles.burgerBtnClick]: menuActive )
```
и потом просто `className={burgerBtnClassname }`